src/Domain/BucketSchema.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace MediaWiki\Extension\Lakat\Domain;
+
+class BucketSchema {
+	public const DEFAULT_ATOMIC = 1;
+	public const DEFAULT_MOLECULAR = 2;
+	public const DEFAULT_PULLREQUEST = 3;
```

Comment:
For now only DEFAULT_ATOMIC and DEFAULT_MOLECULAR

the others are namespaces but not bucket schemata.